src/modules/ie-objects/elasticsearch/elasticsearch.consts.ts

```diff
@@ -202,3 +202,5 @@ export const NEEDS_AGG_SUFFIX: { [prop in IeObjectsSearchFilterField]?: string }
 	[IeObjectsSearchFilterField.GENRE]: 'keyword',
 	[IeObjectsSearchFilterField.TYPE]: 'keyword',
 };
+
+export const ES_DIGIT_PREFIX = '_ç_ç_ç_ç_ç_';
```

Comment:
REQ: please add some comments to further elaborate why this is needed and what it is. For future developers this is just something weird.